src/test/java/com/dev/bootbasic/user/repository/JdbcUserRepositoryTest.java

```diff
@@ -0,0 +1,58 @@
+package com.dev.bootbasic.user.repository;
+
+import com.dev.bootbasic.user.domain.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.JdbcTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@JdbcTest
+@ActiveProfiles("test")
+@Import(JdbcUserRepository.class)
+public class JdbcUserRepositoryTest {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    private User user;
+
+    @BeforeEach
+    public void setup() {
+        setUser();
+    }
+
+    @Test
+    public void createTest() {
+        User savedUser = userRepository.create(user);
+        assertThat(savedUser).isEqualTo(user);
+    }
+
+    @Test
+    public void findByIdTest() {
+        userRepository.create(user);
+        Optional<User> foundUser = userRepository.findById(user.getId());
+        assertThat(foundUser.get()).isEqualTo(user);
+    }
+
+    @Test
+    public void deleteByIdTest() {
+        userRepository.create(user);
+        userRepository.deleteById(user.getId());
+        Optional<User> foundUser = userRepository.findById(user.getId());
+        Assertions.assertTrue(foundUser.isEmpty());
```

Comment:
[nit]
```suggestion
        Assertions.assertThat(foundUser).isEmpty(); 
```
이렇게도 되지 않나요?

Reply:
감사합니다! 반영했습니다
[6ca3777](https://github.com/prgrms-be-devcourse/springboot-basic/pull/798/commits/6ca3777c176fd2e1f88dccd870d60cfc999b7ac3)